docs/configuration/external-dbs.md

```diff
@@ -1,112 +1,110 @@
 # Use external databases
 
-Even thought we deliver postgres as a requirements so
-that you can start Codacy really fast, it's really not
-recommended for a production ready installation.
+Even though we deliver a Postgres instance as part of the Kubernetes cluster so that you can start Codacy really fast, this configuration is really not recommended for a production ready installation.
```

Comment:
Clarified that the Postgres instance is delivered inside the Kubernetes cluster.